.github/workflows/run_chatgpt_examples.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: Install Transformers
         run: |
           cd applications/Chat
-          git clone https://github.com/hpcaitech/transformers
+          git clone --depth 1 --branch v4.30.2 https://github.com/huggingface/transformers
```

Comment:
Is v4.30.2 a stable version? Can we install it from PYPI?

Reply:
This is fixed in 3d5d4ec.